fixchain/fix_and_log_test.go

```diff
@@ -270,11 +281,17 @@ NextToFix:
 }
 
 func TestQueueAllCertsInChain(t *testing.T) {
+	ctx := context.Background()
 	for i, test := range fixAndLogQueueTests {
 		f := &Fixer{toFix: make(chan *toFix)}
+		c := http.Client{Transport: &testRoundTripper{}}
```

Comment:
Similarly here, could just define c as a pointer to an http.Client then you don't have to take the address when passing to client.New below.